installer/scripts/omsadmin.sh

```diff
@@ -445,8 +428,8 @@ onboard_scom()
     is_scom_port_open
     if [ $? -eq 1 ]; then
         echo "Opening port 1270"
-        /opt/omi/bin/omiconfigeditor httpsport -a 1270 < /etc/opt/omi/conf/omiserver.conf > /etc/opt/omi/conf/omiserver.conf_temp
-        mv /etc/opt/omi/conf/omiserver.conf_temp /etc/opt/omi/conf/omiserver.conf
+        $OMI_CONF_EDITOR httpsport -a 1270 < $OMI_CONF_FILE > /etc/opt/omi/conf/omiserver.conf_temp
+        mv /etc/opt/omi/conf/omiserver.conf_temp $OMI_CONF_FILE
```

Comment:
This could be rewritten as:

```
$OMI_CONF_EDITOR httpsport -a 1270 < $OMI_CONF_FILE > ${OMI_CONF_FILE}_temp
mv ${OMI_CONF_FILE}_temp $OMI_CONF_FILE
```

Just another way to do the same thing. I'm not asking for the change, just pointing it out.